From a7fa3c8e8c7b2dab5a704493b7965bbc60f84bed Mon Sep 17 00:00:00 2001
From: Vladimir 'phcoder' Serbinenko <phcoder@gmail.com>
Date: Sun, 22 Jul 2012 16:21:24 +0200
Subject: [PATCH 011/482] 	Fix coreboot compilation.

	* grub-core/term/i386/pc/vga_text.c (grub_vga_text_init): Rename to ...
	(grub_vga_text_init_real): ... this.
	(grub_vga_text_fini): Rename to ...
	(grub_vga_text_fini_real): ... this.
---
 ChangeLog                         | 9 +++++++++
 grub-core/term/i386/pc/vga_text.c | 8 ++++----
 2 files changed, 13 insertions(+), 4 deletions(-)

diff --git a/ChangeLog b/ChangeLog
index 5e54eda..35e76af 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,3 +1,12 @@
+2012-07-22  Vladimir Serbinenko  <phcoder@gmail.com>
+
+	Fix coreboot compilation.
+
+	* grub-core/term/i386/pc/vga_text.c (grub_vga_text_init): Rename to ...
+	(grub_vga_text_init_real): ... this.
+	(grub_vga_text_fini): Rename to ...
+	(grub_vga_text_fini_real): ... this.
+
 2012-07-07  Vladimir Serbinenko  <phcoder@gmail.com>
 
 	* grub-core/Makefile.am: Fix path to boot/i386/pc/startup_raw.S.
diff --git a/grub-core/term/i386/pc/vga_text.c b/grub-core/term/i386/pc/vga_text.c
index d1e4ef9..74c155c 100644
--- a/grub-core/term/i386/pc/vga_text.c
+++ b/grub-core/term/i386/pc/vga_text.c
@@ -167,7 +167,7 @@ grub_vga_text_setcursor (struct grub_term_output *term __attribute__ ((unused)),
 }
 
 static grub_err_t
-grub_vga_text_init (struct grub_term_output *term)
+grub_vga_text_init_real (struct grub_term_output *term)
 {
 #ifdef MODESET
   struct grub_bios_int_registers regs;
@@ -186,7 +186,7 @@ grub_vga_text_init (struct grub_term_output *term)
 }
 
 static grub_err_t
-grub_vga_text_fini (struct grub_term_output *term)
+grub_vga_text_fini_real (struct grub_term_output *term)
 {
 #ifdef MODESET
   struct grub_bios_int_registers regs;
@@ -255,8 +255,8 @@ static struct grub_term_output grub_vga_text_term =
 #else
     .name = "vga_text",
 #endif
-    .init = grub_vga_text_init,
-    .fini = grub_vga_text_fini,
+    .init = grub_vga_text_init_real,
+    .fini = grub_vga_text_fini_real,
     .putchar = grub_vga_text_putchar,
     .getwh = grub_vga_text_getwh,
     .getxy = grub_vga_text_getxy,
-- 
1.8.2.1

